packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -133,27 +184,57 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
         {!isGrantDocument && (
           <div css={buttonsContainer}>
             {canEditResearchOutput && (
-              <Link
-                noMargin
-                href={
-                  sharedResearch({})
-                    .researchOutput({ researchOutputId: id })
-                    .editResearchOutput({}).$
-                }
-                buttonStyle
-                small
-                primary
-              >
-                {editIcon} Edit
-              </Link>
+              <div css={leftButtons}>
+                <Link
+                  noMargin
+                  href={
+                    sharedResearch({})
+                      .researchOutput({ researchOutputId: id })
+                      .editResearchOutput({}).$
+                  }
+                  buttonStyle
+                  small
+                  primary
+                >
+                  {editIcon} Edit
+                </Link>
+              </div>
             )}
             {canDuplicateResearchOutput && duplicateLink && (
-              <Link noMargin href={duplicateLink} buttonStyle small primary>
-                {duplicateIcon} Duplicate
-              </Link>
+              <div css={leftButtons}>
+                <Link noMargin href={duplicateLink} buttonStyle small primary>
+                  {duplicateIcon} Duplicate
+                </Link>
+              </div>
+            )}
+            {!published && !canPublishResearchOutput && (
+              <div css={reviewButton}>
+                <Button
+                  noMargin
+                  small
+                  primary
+                  onClick={() => setDisplayModal(!displayModal)}
+                >
+                  {actionIcon} Ready for PM Review
+                </Button>
+              </div>
             )}
           </div>
         )}
+        {displayModal && (
+          <ConfirmModal
+            title="Output Ready for PM Review?"
+            description="All team members listed on this output will be notified and PMs will be able to review and publish this output."
```

Comment:
since we're already checking if the association is team or working group above to show the text in the toast, couldn't we check here as well? :) 

So we would have `All team members...` or `All working group members...` and then this ticket would address team and working groups already, right?